src/assets/i18n/en.json5

```diff
@@ -2455,6 +2455,8 @@
 
   "form.other-information.institution-entity-name": "Entity",
 
+  "form.other-information.authors-identifier-noma": "NOMA",
```

Comment:
This is not par tof this commit, please remove and place into the correct commit (or create a new one)